po/ru.po

```diff
@@ -244,7 +243,7 @@ msgstr "Усиление входа"
 #: data/ui/pipe_manager_box.ui:209 data/ui/pitch.ui:389
 #: data/ui/presets_menu.ui:28 data/ui/reverb.ui:446 data/ui/rnnoise.ui:239
 #: data/ui/stereo_tools.ui:381 data/ui/stereo_tools.ui:638
-#: src/application_ui.cpp:277 src/application_ui.cpp:265
```

Comment:
Oh I misread, it actually did change something. Albeit not a very interesting change still technically valid.